### DISCUSSION TITLE
Virus Detection

### DISCUSSION THREAD

**Discussion by IT-Support-User** [Romanitho/Winget-AutoUpdate#549] [General]
Hello, 

I have a Virus Detection. 

![Screenshot 2024-01-18 163545](https://github.com/Romanitho/Winget-AutoUpdate/assets/134392782/a647d7f0-ce58-4247-a727-764ec8785ee6)

**Comment by AndrewDemski-ad-gmail-com** [comment]
> Hello,
> 
> I have a Virus Detection.
> 
> ![Screenshot 2024-01-18 163545](https://private-user-images.githubusercontent.com/134392782/297804182-a647d7f0-ce58-4247-a727-764ec8785ee6.png?

That is another instance of already described issue: https://github.com/Romanitho/Winget-AutoUpdate/issues/540

TL;DR: AV does not like LNK files with command line for PowerShell process and that is how the WAU Configurator is launched.
It will stay that way until LNK is replaced with some other method.

**Reply by Romanitho** [reply]
Very true. I couldn't imagine that this method would generate that amount of virus alertes... will be changes soon, as it cannot remain like this...

**Reply by Romanitho** [reply]
https://github.com/Romanitho/Winget-AutoUpdate/releases/tag/v1.19.2-6
Do you still have the issue with the latest beta ?

**Reply by zeehaque** [reply]
That beta worked for me - no issue with AV.
